scripts-available/CDB_Organizations.sql

```diff
@@ -25,6 +25,57 @@ BEGIN
 END
 $$ LANGUAGE PLPGSQL VOLATILE;
 
+-------------------------------------------------------------------------------
+-- Administrator
+-------------------------------------------------------------------------------
+CREATE OR REPLACE
+FUNCTION cartodb._CDB_Organization_Admin_Role_Name()
+    RETURNS TEXT
+AS 'SELECT ''cdb_org_admin''::text || ''_'' || md5(current_database());'
```

Comment:
debugging that will be a pain but the 64 characters limit is hard. 

Should we add a check to CDB_Group_CreateGroup or just leave postgres raise the exception ?